app/src/main/java/com/xmartlabs/template/analytics/AnalyticsManager.java

```diff
@@ -0,0 +1,33 @@
+package com.xmartlabs.template.analytics;
+
+import android.support.annotation.NonNull;
+
+import com.annimon.stream.Stream;
+import com.xmartlabs.base.core.log.analytics.AnalyticTrackable;
+import com.xmartlabs.base.core.log.analytics.AnalyticTracker;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+public final class AnalyticsManager {
```

Comment:
Also if you want you can add something general in the basefragment in order to track screens. Something like:
```java
  protected void trackViewIfNeeded() {
    Optional.ofNullable(getScreenAnalyticsEvent())
        .ifPresent(manager::track);
  }

  @Nullable
  protected AnalyticEvent getScreenAnalyticsEvent(){
    return null;
  };
```